pkg/image/importer/importer.go

```diff
@@ -397,6 +418,7 @@ func importRepositoryFromDocker(ctx gocontext.Context, retriever RepositoryRetri
 			importDigest.Err = err
 			continue
 		}
+		importDigest.Err = calculateImageSize(ctx, repo, importDigest.Image)
```

Comment:
> I'm still concerned because each of those stats could fail, resulting in other failures. We probably *don't* want to fail import if size can't be calculated. We probably don't want to add a fan-out of 20 requests on setup. 

Yes, this is possible. We use NewRetryRepository to make few retries. But fix is necessary for the scheme1.